### PR TITLE
Bug 1999852: bump OVN to ovn21.09-21.09.0-18.el8fdp

### PR DESCRIPTION
**Pull request by dcbw**
Includes fixes for:
- optimizations to loadbalancer logical flow creation in northd (reduced latency and iteration time)
- optimized hashing in northd (reduced latency and iteration time)
- additional northd stopwatches for debugging iteration times
- two northd perf optimizations to avoid multiple identical datapath groups
- optimization to JSON serialization improving all inter-component communication

### CODE REVIEW

**Comment by trozet** [comment]
we do not want the revert for zero snat @flavio-fernandes

**Comment by trozet** [comment]
@dceara FYI

**Comment by flavio-fernandes** [comment]
> we do not want the revert for zero snat @flavio-fernandes

Correct. Let's undo the (partially?)  
https://github.com/ovn-org/ovn/commit/82e241542cbf19a144257a72324bc3278f76ee99 change in u/s ovn and not use this tag.   @putnopvut  agree?

**Comment by dceara** [comment]
> > we do not want the revert for zero snat @flavio-fernandes
> 
> Correct. Let's undo the (partially?)
> [ovn-org/ovn@82e2415](https://github.com/ovn-org/ovn/commit/82e241542cbf19a144257a72324bc3278f76ee99) change in u/s ovn and not use this tag. @putnopvut agree?

@flavio-fernandes I'm not sure it's that straightforward.  We need to see how to do it without reintroducing https://bugzilla.redhat.com/show_bug.cgi?id=1992705

**Comment by dceara** [comment]
@numansiddique FYI

**Comment by dceara** [comment]
@numansiddique @flavio-fernandes @trozet @putnopvut FYI [series to reintroduce all-zero SNAT.](http://patchwork.ozlabs.org/project/ovn/list/?series=260617&state=*)

**Comment by flavio-fernandes** [comment]
> @numansiddique @flavio-fernandes @trozet @putnopvut FYI [series to reintroduce all-zero SNAT.](http://patchwork.ozlabs.org/project/ovn/list/?series=260617&state=*)

lgtm!

File: Dockerfile [hunk 1]
```diff
@@ -33,7 +33,7 @@ RUN yum install -y  \
 	yum clean all
 
 ARG ovsver=2.15.0-28.el8fdp
```
Comment (flavio-fernandes) [inline Dockerfile:35]: @dcbw won't we also need a new OVS to satisfy the call to the recently introduced 'ovsrec_server_has_datapath_table' symbol?

Ref: https://patchwork.ozlabs.org/project/ovn/patch/20210901191304.532.97471.stgit@dceara.remote.csb/

Comment (dceara) [reply Dockerfile:35]: @flavio-fernandes Not really, that's just at compile time and the OVN rpm was built with the correct OVS libs.

Comment (flavio-fernandes) [reply Dockerfile:35]: Ah. TY. Learning something new every day! ;)

**Comment by abhat** [comment]
/override ci/prow/e2e-gcp-ovn
Fails because of: https://bugzilla.redhat.com/show_bug.cgi?id=2000589

File: Dockerfile (hunk 1 above)
Comment (Tongjian-zhang) [reply Dockerfile:35]: > 
how to do that?
i do like this ./configure --with-ovs-source=/root/ovs/openvswitch-2.16.0 --with-ovs-build= /root/ovs/openvswitch-2.6.0
but still has error: undefined reference ""ovsrec_server_has_datapath_table

**Pull request merged**